src/canvg.js

```diff
@@ -2148,6 +2148,9 @@ function build(opts) {
 
       // tween value linearly
       var newValue = p.from.numValue() + (p.to.numValue() - p.from.numValue()) * p.progress;
+      if (this.initialUnits === '%') {
+        newValue *= 100.0; // numValue() returns 0-1 whereas properties are 0-100
```

Comment:
bug with %'s in animations

Reply:
LGTM... (Don't know if you want me to "resolve conversation")